roles/docker-cleanup/files/docker-cleanup.service

```diff
@@ -7,6 +7,7 @@ ConditionPathExists=/etc/docker-gc-exclude-containers
 Type=oneshot
 ExecStartPre=-/usr/bin/docker rm -v docker-gc
 ExecStartPre=/bin/echo "Starting disk usage" ; /bin/df -h
+ExecStart=-/usr/bin/bash -c "/usr/bin/docker rm $(/usr/bin/docker ps -qaf 'status=dead')"
```

Comment:
Ah, that's a good call. May as well not make docker-cleanup-volumes do anything it needn't